src/admin/dashboard/topbar/style.css

```
@@ -0,0 +1,26 @@
+   older plugin-ui build that lacks the Top Bar's responsive variants
+   (md:h-7.5, md:px-2, …). Their base utilities (p-0, h-10) then beat the
+   (0,1,0) copies in plugin-ui's styles.css at every viewport and squash the
+   version badges into oversized ovals. Re-emitting the utilities under the
+   header's ID — (1,1,0) + !important — makes the Top Bar win regardless of
+   what other plugins load. */
+@layer theme, base, components, utilities;
+
+@import "tailwindcss/theme.css" layer(theme);
+
+#tryaura-admin-header {
+  @import "tailwindcss/utilities.css" layer(utilities) important;
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)